src/include/kdbprivate.h

```diff
@@ -210,6 +220,23 @@ struct _Key
 	 */
 	size_t dataSize;
```

Comment:
Also should be, imho, valueSize.

Reply:
Again, was already called this (isn't not even a changed line). Also depending on the decision in #4683 may not even matter. Finally, it may be possible (but definitely more difficult) to use `tree-surgeon` here as well.